pkgs/os-specific/linux/autofs/default.nix

```
@@ -48,9 +48,9 @@ stdenv.mkDerivation rec {
   '';
 
   buildInputs = [ linuxHeaders libtirpc libxml2 libkrb5 kmod openldap sssd
-                  openssl cyrus_sasl rpcsvc-proto ];
+                  openssl cyrus_sasl rpcsvc-proto libnsl ];
 
-  nativeBuildInputs = [ flex bison pkgconf ];
+  nativeBuildInputs = [ flex bison pkgconf libnsl.dev ];
```

```suggestion
  nativeBuildInputs = [ flex bison pkgconf ];
```
Seems to build with NIS support without this too.

---

I just tried it without this and autofs was not able to lookup mounts through NIS, so I think it is indeed required